.travis.yml

```diff
@@ -71,3 +71,21 @@ jobs:
         - df -h
         - docker ps -a
         - for log in $(docker ps -qa | xargs); do docker logs --tail 500 $log; done
+    - # stage name not required, will continue to use `Test`
+      name: "Cross-Site multiple clusters test"
+      services:
+        - docker
+      env:
+        - KIND_VERSION=v0.9.0
+        - METALLB_VERSION=v0.9.6
+      before_script:
+        - sudo snap install yq
+        - source build/travis/configure-travis-ci-xsite.sh
+      script:
+        - go test -v ./test/e2e/xsite/ -timeout 30m
+      after_failure:
```

Comment:
don't you need to stop/delete the kind clusters after the test is finished?

Reply:
Nope, clusters are creating in one-time VM which will be removed